### PR TITLE
[addon-a11y] Fix best practices added in #14736

### PR DESCRIPTION
**Pull request by kaelig**
Issue: in #14736, I added content that had a few issues 😨 

This time, let’s get it right and gather feedback from a few more folks before merging. I'd appreciate a thorough review and opinions on these best practices I'm advocating for.

### CODE REVIEW

**Comment by shilman** [comment]
@tooppaaa @adagar @jsomsanith any chance you can give a thorough review here? will be great to get more a11y process stuff documented for storybook 🙏

**Review by tooppaaa — APPROVED** [review]

**Comment by tooppaaa** [comment]
Thanks for this awesome documentation and guidelines @kaelig !
I would definitely be interested on a chat to help improve a11y experience !

File: addons/a11y/README.md [hunk 1]
```diff
@@ -29,89 +29,132 @@ export default {
...
+    config: {
+      rules: [{ id: 'color-contrast', enabled: false }],
+    },
+  },
+};
+```
 
-    // Instead, override rules 👇
-    // axe-core optionsParameter (https://github.com/dequelabs/axe-core/blob/develop/doc/API.md#options-parameter)
+Tip: clearly explain in a comment why a rule was overriden, it’ll help you and your team trace back why certain violations aren’t being reported or need to be addressed. For example:
```
Comment (jsomsanith) [inline addons/a11y/README.md:92]:
```suggestion
Tip: clearly explain in a comment why a rule was overridden, it’ll help you and your team trace back why certain violations aren’t being reported or need to be addressed. For example:
```

**Comment by jsomsanith** [comment]
Nice improvement. Just a typo ☝️

**Comment by kaelig** [comment]
Thank you very much, and apologies for the first PR riddled with wrong code!

**Pull request merged**